assets/javascripts/modules/toggle.js

```diff
@@ -112,7 +112,7 @@ var toggleEvent = function ($elem) {
       if (event.which === 1 || 32) {
         var $input = $(event.target)
         if ($input.prop('tagName') === 'LABEL') $input = $input.find('input').first()
-        if ($input.prop('tagName') === 'INPUT') {
+        if ($input.prop('tagName') === 'INPUT' && $input.data('target')) {
```

Comment:
it feels like a more robust method would be to check for the `.js-toggle-trigger` class as this is what defines a trigger element. The `target` attribute could be on other inputs.

```
$input.attr('class').indexOf('js-toggle-trigger') >= 0
```